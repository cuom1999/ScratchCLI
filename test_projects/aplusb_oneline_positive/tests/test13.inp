9617089660281 5057339501445