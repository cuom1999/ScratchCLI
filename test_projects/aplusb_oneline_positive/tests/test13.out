14674429161726